avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -752,6 +752,9 @@ public SchemaAndValue toConnectData(org.apache.avro.Schema avroSchema, Object va
   }
 
   private Object toConnectData(Schema schema, Object value) {
+    if (value == null) {
+      return null;
```

Comment:
I know this wasn't there before, but could we add a check that, if the schema is non-null, that it is actually optional? If we see a null here for a required field, we should really be throwing a `DataException`.